src/component.js

```diff
@@ -69,7 +69,7 @@ export default class Component {
         this.listenForNewElementsToInitialize()
 
         if (typeof initReturnedCallback === 'function') {
-            // Run the callback returned form the "x-init" hook to allow the user to do stuff after
+            // Run the callback returned from the "x-init" hook to allow the user to do stuff after
```

Comment:
This is the only error I was looking for actually, I don't think we need the other changes.